easypost/easypost_object.py

```diff
@@ -1,9 +1,10 @@
 import json
+from typing import Any, List, Optional
 
 import easypost
 
 
-def convert_to_easypost_object(response, api_key, parent=None, name=None):
+def convert_to_easypost_object(response: dict, api_key: Optional[str] = None, parent: object = None, name: str = None):
```

Comment:
Shouldn't all of these with `None` as the fallback each be wrapped with `Optional`?

This is true throughout the changes in this PR.